newIDE/app/src/MainFrame/EditorContainers/HomePage/TeamSection/TeamMemberRow.js

```diff
@@ -46,6 +50,23 @@ const TeamMemberRow = ({
 }: Props) => {
   const { isMobile } = useResponsiveWindowSize();
   const gdevelopTheme = React.useContext(GDevelopThemeContext);
+  const iconRestoreTimeout = React.useRef<?TimeoutID>(null);
+  const [emailCopySuccess, setEmailCopySuccess] = React.useState<boolean>(
+    false
+  );
+
+  React.useEffect(
+    () => {
+      if (emailCopySuccess) {
+        if (iconRestoreTimeout.current)
+          clearTimeout(iconRestoreTimeout.current);
+        iconRestoreTimeout.current = setTimeout(() => {
+          setEmailCopySuccess(false);
+        }, 2000);
+      }
```

Comment:
nitpicking: I tend to prefer async methods that allows to write code that looks more sequential:

```suggestion
      (async () => {
        if (!emailCopySuccess) return;
        await delay(2000);
        setEmailCopySuccess(false);
      })();
```

The downside is that can have multiple of these running a the same time... so your timeout ref is more solid in case of quickly copying multiple emails.

Reply:
Good idea, I think the legibility is more important than the robustness of this simple operation